src/api/telemetry/WebSocketWorker.js

```diff
@@ -245,7 +245,6 @@ export default function installWorker() {
     }
```

Comment:
Will try and figure out how to get Webpack to inline Workers so we can do imports etc. Should not block merge right now, it's a no-risk maintenance change we can make later.

Reply:
Some discussion on this here with some potential options - https://github.com/webpack/webpack/discussions/14066

Reply:
This could happen post-merge as it's low risk. It either loads or doesn't.